### PR TITLE
feat(sign up form): add new fields, styles

### PR DESCRIPTION
**Pull request by avramenkop**

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -6,33 +6,33 @@ import { AppComponent } from './app.component';
...
+  declarations: [AppComponent, SignInComponent, SignUpComponent],
   imports: [
     BrowserModule,
     AppRoutingModule,
     BrowserAnimationsModule,
     ReactiveFormsModule,
     ...materialModules,
-    FormsModule
+    FormsModule,
+    MatSelectModule,
```
Comment (otecMuxah) [inline src/app/app.module.ts]: i gusess it shuld go to materialModules

File: src/app/sign-up/sign-up.component.scss [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  width: 100%;
+
+  &__title {
+    font-family: 'impact', sans-serif;
+    font-size: 2.625rem;
+    color: #F8BB15;
+    margin-bottom: 2.5rem;
+  }
+
+  @media (max-width: 25rem) {
```
Comment (otecMuxah) [inline src/app/sign-up/sign-up.component.scss]: not sure it is a best approach to have rem in media query

File: src/app/sign-up/sign-up.component.ts [hunk 3]
```diff
@@ -1,25 +1,45 @@
...
+import { SignUpService } from './services/sign-up.service';
+import { Observable } from 'rxjs';
+import { ICountry } from './models/country.model';
 
 @Component({
   selector: 'app-sign-up',
   templateUrl: './sign-up.component.html',
-  changeDetection: ChangeDetectionStrategy.OnPush
+  styleUrls: ['./sign-up.component.scss'],
+  providers: [SignUpService],
```
Comment (otecMuxah) [inline src/app/sign-up/sign-up.component.ts]: i guess there is no reason to provide it on component level

**Review by otecMuxah — CHANGES_REQUESTED** [review]

File: src/app/sign-up/sign-up.component.scss (hunk 2 above)
Comment (avramenkop) [reply src/app/sign-up/sign-up.component.scss]: updated

File: src/app/app.module.ts (hunk 1 above)
Comment (avramenkop) [reply src/app/app.module.ts]: updated

File: src/app/sign-up/sign-up.component.ts (hunk 3 above)
Comment (avramenkop) [reply src/app/sign-up/sign-up.component.ts]: updated

**Review by otecMuxah — APPROVED** [review]

**Pull request merged**